pkg/ocm/machines.go

```diff
@@ -262,12 +260,7 @@ func (mtl *MachineTypeList) ValidateMachineType(machineType string, multiAZ bool
 	v := mtl.Find(machineType)
 
 	if v == nil {
-		allMachineTypes := strings.Join(mtl.IDs(), " ")
-		allAvailabilityZones := strings.Join(mtl.AvailabilityZones, ",")
-		err := fmt.Errorf("Machine type '%s' not found in availability zones '%s' for region: '%s'\n"+
-			"Available machine type list: %s",
-			machineType, allAvailabilityZones, mtl.Region, allMachineTypes)
-		return err
+		return nil // Replaced not-found error with a preflight in CS (validateZoneSupportInstanceType)
```

Comment:
Note: Reasoning for having this is that the quota check after will cause nil ptrs if it is nil. There's no reason to run the validation if it cannot be found immediately by the CLI